docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+services:
+  db:
+    image: mdillon/postgis:10-alpine #postgres:10-alpine    
+    container_name: postgres01
+    ports:
+      - "5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=postgres
+      - POSTGRES_PASSWORD=postgres
+  nginx:
+    image: nginx:1.13-alpine
+    container_name: nginx01
+    ports:
+      - "80:80"
+    volumes:
+      - ./project:/src
+      - ./config/nginx:/etc/nginx/conf.d
+    depends_on:
+      - web
+  web:
+    build: .
+    container_name: django01
+    #command: bash -c "python manage.py makemigrations && python manage.py migrate && gunicorn interactivemap.wsgi -b 0.0.0.0:80"
```

Comment:
This line can be removed altogether?

I had a 10s pause wondering why it was commented.

Reply:
Removing the comment right? Yeah definitely, forgot I left that there.